src/common/dav/dav.h

```diff
@@ -509,13 +537,30 @@ struct dav_heap_stats {
  * Returns the heap allocation statistics associated  with the
  * DAV object.
  */
-int dav_get_heap_stats(dav_obj_t *pop, struct dav_heap_stats *st);
+int
+dav_get_heap_stats(dav_obj_t *pop, struct dav_heap_stats *st);
 
 struct umem_wal_tx;
 
-uint32_t wal_tx_act_nr(struct umem_wal_tx *tx);
-uint32_t wal_tx_payload_len(struct umem_wal_tx *tx);
-struct umem_action *wal_tx_act_first(struct umem_wal_tx *tx);
-struct umem_action *wal_tx_act_next(struct umem_wal_tx *tx);
+uint32_t
+wal_tx_act_nr(struct umem_wal_tx *tx);
+uint32_t
+wal_tx_payload_len(struct umem_wal_tx *tx);
+struct umem_action *
+wal_tx_act_first(struct umem_wal_tx *tx);
+struct umem_action *
+wal_tx_act_next(struct umem_wal_tx *tx);
+
+/**
+ * Get an evictable zone with sufficient free space within.
+ *
+ * \param[in]		pop		pool handle
+ * \param[in]		flags		zone selection criteria.
+ *
+ * \return id >= 0. Zero indicates non-evictable zone and will be
+ *	returned if no evictable zone can be chosen.
```

Comment:
More non-evictable zones are also planned in the future, or there will be only this one?

Reply:
Its planned for future